bundler/spec/dependabot/bundler/file_fetcher/path_gemspec_finder_spec.rb

```diff
@@ -35,7 +35,7 @@
       let(:gemfile) { bundler_project_dependency_file("path_source", filename: "Gemfile") }
       it { is_expected.to eq([Pathname.new("plugins/example")]) }
 
-      context "whose path must be eval-ed" do
+      context "when the path must be eval-ed" do
```

Comment:
The broader context here is:

> when the file does include a path gemspec when the path must be eval-ed

Maybe this would be nicer?

> when the file does include a path gemspec with a path that must be eval-ed